src/Evolution/Executables/ScalarWave/EvolveScalarWave.hpp

```diff
@@ -112,14 +113,12 @@ class er;
 }  // namespace PUP
 /// \endcond
 
-template <size_t Dim, typename InitialData>
+template <size_t Dim>
 struct EvolutionMetavars {
   static constexpr size_t volume_dim = Dim;
-  // Customization/"input options" to simulation
-  using initial_data_tag = Tags::AnalyticSolution<InitialData>;
-  static_assert(
-      is_analytic_data_v<InitialData> xor is_analytic_solution_v<InitialData>,
-      "initial_data must be either an analytic_data or an analytic_solution");
+
+  using initial_data_list =
+      tmpl::append<ScalarWave::Solutions::all_solutions<Dim>>;
```

Comment:
[Nitpick] I don't think you need to wrap this in an `append`. It's already a list.